app/models/hold.rb

```diff
@@ -4,16 +4,62 @@
 class Hold
   include BibRecord
 
+  PICKUP_LOCATION = {
+    ABINGTON: 'Abington',
+    ACQ_DSL: 'Acquisitions (Law Schools)',
+    ACQUISTNS: 'Acquisitions (UP)',
+    ALTOONA: 'Altoona',
+    BEAVER: 'Beaver',
+    BEHREND: 'Erie',
+    BERKS: 'Berks',
+    BRANDYWINE: 'Brandywine',
```

Comment:
@cdmo do we need double quotes here or not?

Reply:
Because there is a hyphen.. I guess I could have written this as a hash-rocket style. Or does an underscore get converted to a hyphen automatically? i can't remember...

Reply:
oh I see 👍